manim/renderer/opengl_renderer.py

```diff
@@ -272,7 +272,7 @@ def render_mobjects(self, mobs):
 
     def render_render_group(self, render_group):
         shader_wrapper = render_group["shader_wrapper"]
-        shader_program = render_group["prog"]
+        # shader_program = render_group["prog"] Unused?
```

Comment:
Given that the OpenGL renderer is a moving target, I'd rather keep the code here as-is for now.